### ISSUE TITLE
use java languages

### ISSUE THREAD

**Issue by chenjinguang** [armcha/Ribble#4]
I'm not familiar with kotlin now,but i am interested in your mvp + clean-architecture,wish to see this in java code

**Comment by armcha** [comment]
Hi @chenjinguang, sorry but I will not be able to look at it anytime soon.

**Comment by Kessir** [comment]
Hi @chenjinguang don't feel intimidated by Kotlin. If you know Java, you know at least 50% percent of Kotlin. 
Kotlin is the language of the future for android development. It would be a good investment to start learning it now.

**Comment by Kessir** [comment]
@armcha you should probably close this issue since you've addressed it.

**Comment by chenjinguang** [comment]
@Kessir you're right,I've translated this into java language,during this I have learned many things about kotlin,I think it's also a good way to learn kotlin,maybe I will use kotlin in my company's projects soon,thank you :)
